app2/src/main/java/com/example/recipebox/MainActivity.kt

```diff
@@ -21,23 +19,31 @@ class MainActivity : ComponentActivity() {
                 Surface(
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colors.background
+
                 ) {
-                    Greeting("Android")
```

Comment:
You can remove the default greeting generated by IntelliJ since you will likely need to use a scaffold to create that top bar

Reply:
Good